tuf/client/updater.py

```diff
@@ -1493,16 +1493,30 @@ def _get_metadata_file(self, metadata_role, remote_filename,
         # number, the new metadata is safe to parse.
         try:
           metadata_spec_version = metadata_signable['signed']['spec_version']
-          metadata_spec_major_version = int(metadata_spec_version.split('.')[0])
-          code_spec_major_version = int(tuf.SPECIFICATION_VERSION.split('.')[0])
-
+          metadata_spec_version_split = metadata_spec_version.split('.')
+          metadata_spec_major_version = int(metadata_spec_version_split[0])
+          metadata_spec_minor_version = int(metadata_spec_version_split[1])
+
+          code_spec_version_split = tuf.SPECIFICATION_VERSION.split('.')
+          code_spec_major_version = int(code_spec_version_split[0])
+          code_spec_minor_version = int(code_spec_version_split[1])
+ 
           if metadata_spec_major_version != code_spec_major_version:
             raise tuf.exceptions.UnsupportedSpecificationError(
                 'Downloaded metadata that specifies an unsupported '
                 'spec_version.  This code supports major version number: ' +
                 repr(code_spec_major_version) + '; however, the obtained '
                 'metadata lists version number: ' + str(metadata_spec_version))
 
+          #report to user if minor versions do not match, continue with update
```

Comment:
The comment suggests `!=` but the code and the log message `>`. Which one is it?

Reply:
I updated the operator and message to report any mismatch in the minor version.